domain/src/main/java/tachiyomi/domain/library/service/LibraryPreferences.kt

```diff
@@ -26,6 +26,8 @@ class LibraryPreferences(
         LibrarySort.Serializer::deserialize,
     )
 
+    fun currentRandomSortSeed() = preferenceStore.getInt("library_random_sort_seed", 0)
```

Comment:
Default of zero, but will be changed when the option is selected, so the seed should never actually be read as zero (though it's not a problem if it is)